cobigen/cobigen-core/src/test/java/com/devonfw/cobigen/unittest/config/upgrade/ContextConfigurationUpgraderTest.java

```diff
@@ -145,6 +148,7 @@ public void testCorrectUpgrade_v2_1_TO_LATEST() throws Exception {
    * @throws Exception test fails
    */
   @Test
+  @Ignore
```

Comment:
Would be nice if there is a comment/hint on why these tests were ignored and on which ticket these should be activated